packages/@vuepress/theme-default/global-components/CodeGroup.vue

```diff
@@ -65,12 +76,18 @@ export default {
   .theme-code-group__nav {
     margin-bottom: -35px;
     background-color: $codeBgColor;
-    padding-bottom: 22px;
+    padding-bottom: 5px;
```

Comment:
taking this out causes the bottom to get clipped on some of the code group tabs:

![Screen Shot 2020-10-01 at 11 17 56 AM](https://user-images.githubusercontent.com/16292606/94835846-d7e6a100-03d7-11eb-8397-8df21ba87bed.png)